app/plugins/misc/meme.py

```
@@ -0,0 +1,31 @@
+        await reply_and_delete(message, "Reddit API credentials are not configured. Please set them in `extra_config.py`.")
+        return
+
+    reddit = asyncpraw.Reddit(
+        client_id=extra_config.REDDIT_CLIENT_ID,
+        client_secret=extra_config.REDDIT_CLIENT_SECRET,
+        user_agent=extra_config.REDDIT_USER_AGENT,
+    )
+    try:
+        subreddit = await reddit.subreddit("memes")
+        hot_memes = [meme async for meme in subreddit.hot(limit=100)]
+        random_meme = random.choice(hot_memes)
```

Ruff (0.14.0)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)